library/Cube/Ido/IdoHostStatusCube.php

```diff
@@ -91,6 +91,7 @@ public function prepareInnerQuery()
 
         $view->applyFilter($this->getMonitoringRestriction());
 
+        $select = $view->getQuery()->getSelectQuery();
```

Comment:
We don't need this twice, do we? 😆

Reply:
Oof, how did that happen?
Sorry, I'll pay better attention to the diffs next time!